source/reference/supported-server-version.txt

```diff
@@ -40,19 +40,50 @@ destination MongoDB server versions.
 .. list-table:: 
    :header-rows: 1
    :stub-columns: 1
-   :widths: 40 30 30 
+   :widths: 30 14 14 14 14 14
```

Comment:
Just a heads up, with the older version support feature branch being merged, we converted this table into an include with the updated version compatibilities.

Reply:
Refactored